### DISCUSSION TITLE
How to modify a gene annotation in a GTF file

### DISCUSSION THREAD

**Discussion by Dalhte** [NBISweden/AGAT#410] [Q&A]
Hello there
I'm currently running single-cell RNA seq analysis on RN7 rattus genome annotation. I used to use RN6 before, and I noticed a somewhat annoying difference concerning one of the major genes expressed in my tissue: The annotation of the nearby gene overlaps now (a novel transcript not present in RN6) with the 3UTR of the gene of interest, and it seems to prevent mapping of the reads by cell ranger. As the overlapping transcript is not expressed in my tissues, I would like to remove if from the GTF RN7 file I'm using, but I don't know how to do that. If somebody can give me clues, it would be nice :)
Best
David

**Comment by Juke34** [comment]
Hi
If you know the mRNA ID you want to remove you might use `agat_sp_filter_feature_from_kill_list.pl`

**Reply by Dalhte** [reply]
Thanks a lot ! 
Best
D

**Comment by Dalhte** [comment]
Hello
I'm sorry to bother you again !
What you propose work fine but I'm still having a problem after having removing the problematic transcript. I would like also to modify the value of end of the parent gene feature. Is it possible to use AGAT to edit a value for on feature ?
Best 
David

**Reply by Juke34** [reply]
To modify a feature you can either use a genome browser like webapollo and save the new annotation, or do it manually opening the file and changing the end column of the corresponding feature.
I do not remmener by heart but if you modify e.g. the gene end position and you process the file with AGAT it will propagate that information to the underlying features (e.g. mRNA, exons, CDS, UTRs).  Or it was by modifying the exon and then it propagates to the UTRs, CDS, mRNA, genes. Maybe it works on both ways... you should check and let me know.

**Reply by Dalhte** [reply]
Thanks !
Opening the gtf was the most simple in the end. Never succeed to launch webapollo (impossible to force using java1.8). It's sad because this tool seems very net !
Thanks again
David
